presentations/en/2019-07-xx-Mandate.md

```diff
@@ -1,47 +1,128 @@
-# IT Strategy
+# Mandate
+
+## IT Strategy
+
+2019-07-xx
+
+---
 
 ## Vision
 
-### Future Testimonials
+### Image of the future
+
+The [Canadian Experience](https://github.com/sara-sabr/ITStrategy/blob/master/Strategies/HumanDevelopmentLifeCycle.md) is when Canadians are benefiting seamlessly of the federal government services through modern technologies.
+
+--
+
+### Testimonials
```

Comment:
I don't understand this section and we probably need user research to confirm but my gut feeling tells me this is going to be common.

Reply:
I put it with the vision, I don't know..  Take it out?

Reply:
I think we may be combining two different things here? A presentation in my mind is a message to a specific audience.

Having our mandate page is a go-to place for a quick understanding of what we're doing.